application/Program.cs

```diff
@@ -12,7 +12,7 @@ public static void Main(string[] args)
 
 		    using (var amz = new AmazonSynchronizer(new Configuration()))
 		    {
-                amz.WithInterval(TimeSpan.FromMinutes(1))
+                amz.WithInterval(TimeSpan.FromMinutes(5))
```

Comment:
I think I get it now (read over it some more).  Can you change it so that as soon as it finishes with one pass it immediately starts the next without waiting in between?  When this is deployed, I want it to just run forever, looping over all products forever without ever taking any breaks.